css/style.css

```diff
@@ -0,0 +1 @@
+
```

Comment:
You have successfully added a style sheet to your project 👌, please make sure to use CSS, CSS makes the front end of a website shine 😎 and it creates a great user experience, u can use it also to arrange the parts of the website.
More information here 👉 [https://www.devopsschool.com/blog/what-is-css-and-its-importance-in-web-development/](https://github.com/Abdelaziz-Ali/Review-for-SurveyFormHTMLproject/pull/1/files/url).